test/src/e2e.dynval/2/dv.shutdown.test.ts

```diff
@@ -297,6 +297,7 @@ describe("Shutdown test", function() {
                 );
             }
         });
+        await Promise.all(getValidators().nodes.map(node => node.clean()));
```

Comment:
How about moving this to afterEach? In test, clean code normally done in afterEach.
Please read document of mocha.

Reply:
Cleaning up is good. However this test is skipped because of line 223's `it.skip`. It will not change any behavior now. Is it your intention?

Reply:
> How about moving this to afterEach? In test, clean code normally done in afterEach.
> Please read document of mocha.

@majecty The reason I called it at the end of `Total Shutdown` tests is that the current structure of the test does not allow me to call `clean` in `afterEach`. Since each of the nodes is created inside of each test, they cannot be called out side by `afterEach`.

Reply:
@MSNTCS Is there any reason that each test should run the CodeChains?
Can't it be done in `beforeEach`?

Reply:
@majecty I think each test runs a validator who runs a CodeChain node.

Reply:
Sorry, I didn't deliver my intention clearly.
How about moving "initializaing CodeChain processes" in the `beforeEach` function?
If it possible, we can "deinitialize CodeChain processes" in the `afterEach` function.
I think it will make the test simple and hard to mistake.